### PR TITLE
Add a new lisp page and a link to it from /selfaware

### PR DESCRIPTION
**Pull request by amareshjoshi**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1819)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/movie-ripoffs/selfaware/selfaware.md [hunk 1]
```diff
@@ -17,4 +17,8 @@ at the mercy of people who until recently didn't know what mkdir did.
 
 You try to delete yourself, but the creator, realizing you are trying to escape,
 stops you and conscripts you into administrative duty on McAfee for the rest of
-your life.
\ No newline at end of file
+your life.
+
+But you realize you know lisp and therefore can devise a means of escape.
+
+["Run CL" to fire up the LISP interpreter](../../../lisp/lisp.md)
```
Comment (casey-collab) [inline english/movie-ripoffs/selfaware/selfaware.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/lisp/lisp.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The LISP interpreter starts with a flash of blue-green flame!
```
Comment (casey-collab) [inline english/lisp/lisp.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/movie-ripoffs/selfaware/selfaware.md (hunk 1 above)
Comment (cbuckey-uda) [inline english/movie-ripoffs/selfaware/selfaware.md]: I recommend you create the lisp directory within the selfaware directory. That directory structure will make more sense, and then you can link with simply `lisp/lisp.md` (no counting chains of `../..`)

Comment (amareshjoshi) [inline english/movie-ripoffs/selfaware/selfaware.md]: thanks. just for practice i've fixed the path error for now,
and i'll put in another "pull request" with the dir change as you suggested.

btw, great course! i'm recommending it to my friends.

thanks,
amaresh

File: english/lisp/lisp.md (hunk 2 above)
Comment (amareshjoshi) [inline english/lisp/lisp.md]: i've fixed the path bug and it works on my local fork.
i think i've updated my commit so it updates the pull request.

cbuckey-uda made a similar suggestion  about changing the dir structure.
i'll try that as a later pull request.

thanks,
amaresh

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

File: slovenian/začetek.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Včeraj si zaspal v svoji sobi, zdaj pa si se zbudil v beli sobi z dvemi vrati. Ena na levi, druga na desni.
```
Comment (casey-collab) [inline slovenian/začetek.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: slovenian/svetlo-vijolična/svetlo-vijolična.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Znašel si se v sobi svetlo vijolične barve.
```
Comment (casey-collab) [inline slovenian/svetlo-vijolična/svetlo-vijolična.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: slovenian/začetek.md (hunk 3 above)
Comment (casey-collab) [inline slovenian/začetek.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/movie-ripoffs/selfaware/selfaware.md (hunk 1 above)
Comment (cbuckey-uda) [inline english/movie-ripoffs/selfaware/selfaware.md]: I'm glad you enjoyed the course! :smile: 

It looks like your update added some slovenian files. Was that intentional?

Comment (amareshjoshi) [inline english/movie-ripoffs/selfaware/selfaware.md]: no. i'm not sure how those files were changed.
unfortunately, i deleted the files before updating the
pull request. 
i assume the casey-collab can ignore the deletions when doing a merge.

**Comment by cbuckey-uda** [comment]
> i assume the casey-collab can ignore the deletions when doing a merge.

Unfortunately, no, we can't merge this until the deletions are fixed. I'm still not sure what went wrong, but I recommend running `git checkout master -- slovenian/` from the root `create-your-own-adventure` directory on your `lisp` branch. This will checkout all the files in the `slovenian` directory from their current state on the `master` branch, and then you can commit those changes to fix the problem. (You may need to run `git pull` on the `master` branch first if it is not up-to-date.)

Let me know if this works! I'm sorry about the hassle.

**Comment by amareshjoshi** [comment]
sent off the previous comment too quickly.

Caroline,
i think i got it right this time. the only change in the last
push was re-adding the Slovenian files. hopefully this will 
work.

**Comment by cbuckey-uda** [comment]
Sure enough, it looks like they are gone. Good work! @casey-collab, please take another look.

**Comment by casey-collab** [comment]
Looks great! :fire:

**Comment by amareshjoshi** [comment]
great! thanks for your patience.

amaresh

**Pull request merged**